jib-gradle-plugin/src/integration-test/resources/projects/default-target/build.gradle

```diff
@@ -16,6 +16,7 @@ dependencies {
 
 jib {
     container {
+        entrypoint = ['foo', 'bar', 'baz']
```

Comment:
These cause the integration tests to fail, since the integration tests actually try to run the built container and 'foo' isn't an executable. For example, for `testBuildTar_simple`:

```
java.lang.RuntimeException: Command 'docker run simpleimage:gradle3455722031277561' failed: docker: Error response from daemon: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"foo\": executable file not found in $PATH": unknown.
time="2018-08-29T14:41:10-04:00" level=error msg="error waiting for container: context canceled"
```

Reply:
To go off of what @coollog said, maybe it'd be a good idea to put this in a separate test that doesn't actually run the image and just inspects it, but actually runs the images generated using `mainClass` and `jvmFlags`.